docs/agents/sematext-agent/jmx-attaching-containers.md

```diff
@@ -0,0 +1,21 @@
+title: Automatic attaching to Java processes in container environments
+description: Sematext Agent can automatically attach to Java processes (Solr, Kafka, Zookeeper...) and expose JMX port required for monitoring from within the service itself
+
+When you assign a `MONITORING_TOKEN` to a Java service (e.g. via env variable), Sematext Agent will examine the process to see whether it can be
+monitored right away (i.e check if JMX ports are already exposed). If it can, Sematext Agent will start its monitoring. However, if it can't, Sematext Agent
```

Comment:
monitored right away (i.e check if JMX ports are already exposed). If it can, Sematext Agent will start its monitoring. However, if it can't, Sematext Agent
=>
monitored right away by checking if JMX ports are already exposed. If it can, Sematext Agent will start monitoring it. However, if it can't, Sematext Agent